frontend/src/App.js

```diff
@@ -1,19 +1,9 @@
-import {useState} from "react";
-import {postMinutes} from "./service/timer-api-service";
+import PumpAction from "./components/PumpAction";
 
 function App() {
 
-    const [minutes, setMinutes] = useState("")
-
-    const handleClick = () => {
-        postMinutes(minutes).then(response => console.log(response))
-    }
-
     return (
-        <form>
-            <input onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
```

Comment:
I think it is best-practice to always add the input type to the html element and a label

Reply:
Thanks for the advice :)